.gitignore

```diff
@@ -29,3 +29,22 @@ yarn-error.log
 /node_modules
 .env
 docker-compose.override.yml
+
+
+# Swap
+[._]*.s[a-v][a-z]
```

Comment:
You can set a global .gitignore for your session:
```
~/.gitignore_global
*~
.DS_Store
```

Add the following to your `$HOME/.gitconfig`
```
[core]
	editor = vim
	excludesfile = /Users/albertwang/.gitignore_global
```

Reply:
Addressed https://github.com/brave-intl/publishers/pull/1396/commits/754d9c0c9e09649452942f9dd7abff409cc69ef8